src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -417,7 +421,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         const auto hr = wil::ResultFromCaughtException();
 
         winrt::hstring failureText{ fmt::format(std::wstring_view{ RS_(L"ProcessFailedToLaunch") },
-                                                gsl::narrow_cast<unsigned long>(hr),
+                                                fmt::format(_errorFormat, hr),
```

Comment:
Instead of defining `_errorFormat` as a global constant it's probably easier to understand to write:
```cpp
static std::wstring formatResultCode(HRESULT hr)
{
    return fmt::format(L"{0} ({0:#010x})", hr);
}
```

But is there any specific reason you didn't just put this format specifier into the .resw strings (not saying this is wrong - just curious)?

Reply:
![image](https://user-images.githubusercontent.com/189190/133853902-8d05ee88-f738-47c3-a37e-1ff5085d74a2.png)

Reducing the number of times we make the localizers have to localize :smile: